recipes/angelscript/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import CMake, ConanFile, tools
+
+import os
+
+
+class AngelScriptConan(ConanFile):
+    name = "angelscript"
+    license = "zlib"
```

Comment:
```suggestion
    license = "Zlib"
```
https://spdx.org/licenses/Zlib